source/parser/types.civet

```diff
@@ -675,3 +675,8 @@ export type VoidType = ASTLeaf & type: "VoidType"
 export type TypeLiteralNode = ASTLeaf | VoidType
 
 export type ThisAssignments = [string, ASTRef][]
+
+export type WithClause =
+  type: "WithClause"
+  children: Children
+  targets: [WSNode, ExpressionNode][]
```

Comment:
I think this needs to be added to something... `OtherNode`?

Reply:
Currently it's consumed in the parse phase so doesn't end up in the final tree.